tests/foreman/ui/test_templatesync.py

```diff
@@ -39,6 +39,8 @@ def test_positive_import_templates(session, templates_org, templates_loc):
 
     :id: 524bf384-703f-48a5-95ff-7c1cf97db694
 
+    :bz: 1778181, 1778139
```

Comment:
Add the Bugzilla details in the QA whiteboard

Reply:
Done!

Reply:
Added for all BZs here!